### DISCUSSION TITLE
请问 Swipe 轮播组件的swipeTo可以瞬间切换吗？

### DISCUSSION THREAD

**Discussion by speedhigh** [youzan/vant#9582] [Q&A]
在项目中，我想在某个特定的情况下瞬间切换到第一张图片，但是swipeTo是需要滚动时间的，求方案

**Comment by chenjiahan** [comment]
```js
swipeTo(0, { immediate: true })
```

![image](https://user-images.githubusercontent.com/7237365/135083635-e686f61c-8373-4ab6-9e83-7fda045af911.png)
